docs/blockchain/contracts/nft-contract/nft-actions/mknftofr.a.experimental.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'mknftofr.a'
+order: 37
+
+---
+
+# mknftofr.a
+
+Make an offer on an Uniq.
+
+## Technical Behavior
+
+The offer should be done at the time which is in the range of trading window of the NFT factory from which the NFT was issued.
+
+The action stores the offer to `nftoffer.a` table with the specified arguments. The new offer ID is read from `next.nftofr` table whose `value` field is then incremented.
+
+`eosio.nftram` pays RAM usage.
+
+The offered price will be transferred to `eosio.nftofr` account and will be kept until the offer is either accepted or cancelled.
+
+An account will not be able to make offers on the same Uniq that they already made.
+
+An account will not be able to make more offers on Uniq than `max_active_offer_per_user` of `offercfg.a`, which records the global configurations.
```

Comment:
See the code
https://github.com/ultraio/eosio.contracts/blob/master/contracts/eosio.nft.ft/src/nft.buy_offer.cpp#L131

Reply:
update to `An account will not be able to make Uniq offers more than or equal to`

Reply:
What I want to say here is that , from the code https://github.com/ultraio/eosio.contracts/blob/master/contracts/eosio.nft.ft/src/nft.buy_offer.cpp#L131
```
ASSERTION_CHECK(row.factory_ids.size() + row.nft_ids.size() < config.max_active_offer_per_user, buyer_already_made_max_number_of_offers);
```
So max_active_offer_per_user applies to the sum of offers made on Uniqs and offers on factories.

Reply:
So the description should be
An account will not be able to make more offers on either factories or Uniqs

Reply:
I see, updated to `An account will not be able to make offers if their total offers (including Uniq and Uniq factory) is more than or equal to` to be clearer

Reply:
Please see the above comment for `mkfctofr.a` action case.